tinytuya/core.py

```diff
@@ -820,7 +820,7 @@ def assign_dp_mappings( tuyadevices, mappings ):
 
 class XenonDevice(object):
     def __init__(
-            self, dev_id, address=None, local_key="", dev_type="default", connection_timeout=5, version=3.1, persist=False, cid=None, node_id=None, parent=None, connection_retry_limit=5, connection_retry_delay=5, port=TCPPORT # pylint: disable=W0621
+            self, dev_id, address=None, local_key="", dev_type="default", connection_timeout=5, version=3.1, persist=False, cid=None, node_id=None, parent=None, connection_retry_limit=5, connection_retry_delay=5, port=TCPPORT, max_simultaneous_dps=0 # pylint: disable=W0621

```

Comment:
Should caching be an option (that can be turned off) in addition to persist=True?

Also, we should break up the super long line with some \n love.

Reply:
I don't see a point in disabling it.  Cached data is only returned with `.cached_status()` so if you don't want cached data then just call `.status()`.  Enabling it without `persist=True` is going to cause the cached state to not match the device state if someone presses a button on the device or makes a change in the SmartLife app.